app/swagger-doc/requests/TodoBody.yml

```diff
@@ -0,0 +1,9 @@
+components:
+    requestBodies:
+        TodoBody:
+        description: A JSON object containing todo information
+        required: true
+        content:
+            application/json:
+                schema:
+                    $ref: '#/components/schemas/Todo'
```

Comment:
are you sure? schemas/Todo should contain e.g. `createdAt` but request shouldnt contain `createdAt`